### DISCUSSION TITLE
pushrelay-mast1.your.org cleanup

### DISCUSSION THREAD

**Discussion by shleeable** [mastodon/mastodon#17424] [General]
Hey everybody,
I've noticed lots of errors in the logs and sidekiq dead queue related to `pushrelay-mast1.your.org` for almost 2 years now.

![image](https://user-images.githubusercontent.com/17537000/152107546-d6b8b2a0-7641-491a-af16-4f7af7fd1dd4.png)

----

So @Gargron and I have made two commands to clean that spam from the sidekiq dead queue and error logs in general.

Note: if you know who runs/ran `pushrelay-mast1.your.org`, maybe give them a message and see if it's in any way required any more.

----
On the irb console - `docker-compose exec web bundle exec rails console` on docker

`Web::PushSubscription.where('endpoint LIKE ?', 'https://pushrelay-mast1.your.org/%').in_batches.delete_all`
and
`Sidekiq::DeadSet.new.select {_1.display_class == "Web::PushNotificationWorker"}.map(&:delete)`

should resolve the problem of this dead push relay.

**Comment by vmstan** [comment]
This worked to rid some `ntfy.sh` messages.
